### ISSUE TITLE
Permissive icon fonts

### ISSUE THREAD

**Issue by Vekseid** [elkarte/Elkarte#2430]
To help with hunting down images. Sticking with CC0/CC-BY, MIT, and BSD-licensed images. Also want the fonts to 'blend in' with Font Awesome, so things line linecons don't work well.

Fontelico: https://github.com/fontello/fontelico.font  
- Has a smiley set. Have some thoughts on how to incorporate that, but it's a rather major change.
- Otherwise, mostly nifty spinners and that crown icon

Brandico: https://github.com/fontello/brandico.font
- Some brand icons for social networks and such.

Zocial: https://github.com/smcllns/css-social-buttons
- Still more social icons

Octicons: https://github.com/github/octicons
- A few gems in here.

Maki: https://github.com/mapbox/maki
- Some nice ones here, too.

**Comment by Spuds** [comment]
Honestly some of those look rubbish to me :innocent: 

This is a nice set -- https://materialdesignicons.com/

Changing the smiley code would probably be a disaster IMO

**Comment by Vekseid** [comment]
Mostly just gathering a reference list.

And yeah, I was planning on doing something special on my own just for my own forums, regarding smileys. >_>

**Comment by emanuele45** [comment]
I'm a bit lost with icons as usual, is this list useful for 2.0?

**Comment by Spuds** [comment]
Yes ... we still have a few icons to move to SVG and I'd like to consolidate on the material design  in 2.0 (right now we are a mixture of FA and icomoon.  May be able to have an SVG smiley set as well.  So I'll move this to 2.0
